extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -165,6 +166,10 @@ export class InstrumentationService {
       if (this.isTimerSpecValid_(config['timerSpec'])) {
         this.createTimerListener_(listener, config['timerSpec']);
       }
+    } else if (eventType === AnalyticsEventType.TIMEOUT) {
```

Comment:
Lets add `immediate` property to `timer` trigger. I think that would be simpler to understand, code and document.